fractals/sunqua-peak/index.md

```diff
@@ -127,40 +127,40 @@ The boss has multiple phases themed around the different elements air, water, fi
 
-She will keep dashing around the arena and mixing and matching her abilities discribed above until she dashes to the middle of the arena. The only difference with being in the middle of the arena is that during the split phase instead of pulsing AoE's being around the boss lasers appear that will converge once the player specific AoE's explode. Be careful when walking back into the boss as these lasers will converge on her in two waves and deal heavy damage.
+She will keep dashing around the arena and mixing and matching her abilities described above until she dashes to the middle of the arena. The only difference with being in the middle of the arena is that during the split phase instead of pulsing AoE's, lasers appear that will converge once the player specific AoE's explode. Be careful when walking back into the boss as these lasers will converge on her in two waves and deal heavy damage.
 
 From this point onward the boss will mostly stay in the center of the arena, dashing out only to instantly dash back in, or only staying on the wall for the 5 waves of laser attacks.
 
 #### Sorrow<Label>66%-33%</Label>
 <Message>
 Any AoE's on the floor prior to her hitting 66% will still last the full duration so be careful when walking back to the center
 </Message>
-Upon hitting 66% the boss will stop any current animation and fizzle any current attack and dash towards the center of the arena. She will then instantly start channeling and pulsing out AoE's in a set ring pattern. Where the first ring is close, second middle and third far from the boss. However no AoE's will spawn directly next to the boss so staying in melee range is safe. There are also small safe zones between the AoE's.
+Upon hitting 66% the boss will stop any current animation and fizzle any current attack and dash towards the center of the arena. She will then instantly start channeling and pulsing out AoE's in a set ring pattern. Where the first ring is done, second is in the middle and third is far from the boss. However no AoE's will spawn directly next to the boss so staying in melee range is safe. There are also small safe zones between the AoE's.
```

Comment:
"where the first ring is done" this doesnt make sense, the original "close" as opposed to "done" is correct. Its trying to describe the 3 different ranges of the AoE's spawning in a ring pattern, one being a far ring, one being in the middle and one being close.